data/mods/Xedra_Evolved/items/inventor/melee.json

```diff
@@ -0,0 +1,74 @@
+[
+  {
+    "type": "enchantment",
+    "id": "WOLF_SLEDGE",
+    "has": "WIELD",
+    "condition": "ALWAYS",
+    "values": [ { "value": "ITEM_DAMAGE_BASH", "add": { "u_val": "skill_level", "skill": "deduction" } } ]
+  },
+  {
+    "type": "TOOL",
+    "id": "mace_magic",
+    "name": { "str": "wolf sledge" },
+    "description": "A long, solid, crafted steel haft, and a bulky, comma-like head.  It feels really massive, but it is surprisingly easy to swing, and you will do more damage as your level increases.  You can make it return by speaking its command word while throwing.",
+    "weight": "2150 g",
+    "volume": "1750 ml",
+    "longest_side": "130 cm",
+    "color": "dark_gray",
+    "symbol": "/",
+    "looks_like": "sledge",
+    "material": [ "steel" ],
+    "techniques": [ "WBLOCK_1", "SWEEP", "BRUTAL" ],
+    "flags": [ "SHEATH_SPEAR", "DURABLE_MELEE", "ALWAYS_TWOHAND" ],
+    "weapon_category": [ "GREAT_HAMMERS" ],
+    "bashing": 27,
+    "relic_data": { "passive_effects": [ { "id": "WOLF_SLEDGE" } ] },
+    "to_hit": { "grip": "solid", "length": "long", "surface": "any", "balance": "neutral" },
+    "use_action": { "type": "cast_spell", "spell_id": "mace_pipe_magic_throw", "no_fail": true, "level": 0, "need_wielding": true }
+  },
+  {
+    "id": "combatsaw_spear_off",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": { "str": "chainsaw spear (off)", "str_pl": "chainsaw spears (off)" },
+    "description": "A hand-made lightened engine, with chain blades on the long stick with baterry tank inside.  The only reason it works and do not fall apart is because your calculations can't be wrong.",
```

Comment:
```suggestion
    "description": "A hand-made lightened engine, with chain blades on a long stick.  The only reason it works without falling apart is because your calculations can't be wrong.",
```